deployment/ccip/changeset/globals/ocr3.go

```diff
@@ -34,7 +34,7 @@ var (
 		DeltaProgress:               120 * time.Second,
 		DeltaResend:                 30 * time.Second,
 		DeltaInitial:                20 * time.Second,
-		DeltaRound:                  15 * time.Second,
+		DeltaRound:                  2 * time.Second,
```

Comment:
Is that a good idea for Commit to be that fast? That's gonna increase the number of reports generated and reduce that batching capability, no? If we need 3 rounds to create CommitReport it means that in most cases CommitReport would be produced every 6 seconds? Not much time to gather messages for batch. 
That would probably lead to not very efficient batching and higher gas utilization

cc @RensR @matYang

Reply:
Bumped commit round time to 30s on non-ETH and 90s on ETH. It means that we'd produce a commit report every 60s (on average) on non-ETH; assuming a block time of ~2s this is like 30ish blocks, which should be a better batching opportunity. On ETH, a report would be produced every 180s or 15 blocks.